### DISCUSSION TITLE
Achievement Icons could be more smaller

### DISCUSSION THREAD

**Discussion by rootcf** [community/community#41974] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I think the achievement icons on the profile could be a little smaller. When someone has multiple achievements, it doesn't look as nice on their profile if they're too big. In my opinion, they could be a little smaller

# Example
This is the current size of the achievements.
![image](https://user-images.githubusercontent.com/56794915/208237557-6f76dd99-5003-45b8-82f8-bbdf255bb014.png)

And these are the achievements that I have resized as an example.
![image](https://user-images.githubusercontent.com/56794915/208237590-2a7e13dc-4445-4a67-9158-44448b0df319.png)

**Comment by jfullstackdev** [comment]
why would you want that to be smaller ?

**Reply by rootcf** [reply]
I don't think it looks aesthetically pleasing in terms of design. Most of the materials on the profile are small (text, repo boxes, descriptions). While everything on the profile looks small and comfortable enough when looking straight at it, the sudden appearance of massive icons when looking down a little bit disrupts the aesthetics a bit. It's like they were added with an extension. Making them a little smaller could make them more compatible with other materials and look more comfortable in terms of design. Of course, it doesn't have to be like that, these are just my own ideas.

**Comment by rootcf** [comment]
In order to better explain, let me give an example from Linus Torvalds' profile.

## Original
![!linus](https://user-images.githubusercontent.com/56794915/208247349-2a89d7b7-a20b-4c94-9527-6097425f8fc5.PNG)
<br>
## With small icons
![!linus2](https://user-images.githubusercontent.com/56794915/208247365-86db6856-4cfe-4bdc-9c41-a40e4b41070a.PNG)
<br>
Which one looks more comfortable and aesthetically pleasing to the eye in your opinion?
